docs/getting-started.md

```diff
@@ -166,6 +166,8 @@ kubectl -n argo port-forward deployment/argo-server 2746:2746
 
 Then visit: http://127.0.0.1:2746
 
+See the [Argo Server documentation](./argo-server.md) for config options.
```

Comment:
I think adding something like this would be helpful:
```suggestion
See the [Argo Server documentation](./argo-server.md) for config options. Such as Auth Options, Managed Namespace deployments, etc....
```
Pick what you think is most important.

Reply:
Pushed. I just added auth and namespace deployments, since base-href seems like a less common item for someone to need in a getting-started guide.